kubemarine/procedures/restore.py

```diff
@@ -135,19 +136,25 @@ def restore_thirdparties(cluster):
 
     install.system_prepare_thirdparties(cluster)
 
-
+    
 def import_nodes(cluster):
     for node in cluster.nodes['all'].get_ordered_members_list(provide_node_configs=True):
-        node['connection'].put(os.path.join(cluster.context['backup_tmpdir'], 'nodes_data', '%s.tar.gz' % node['name']),
-                               '/tmp/kubemarine-backup.tar.gz')
-        cluster.log.debug('Backup \'%s\' uploaded' % node['name'])
-
-    cluster.log.debug('Unpacking backup...')
-    result = cluster.nodes['all'].sudo(
-        'chattr -i /etc/resolv.conf; sudo tar xzvf /tmp/kubemarine-backup.tar.gz -C / --overwrite && sudo chattr +i /etc/resolv.conf')
-    cluster.log.debug(result)
-
-
+        node_backup_file = os.path.join(cluster.context['backup_tmpdir'], 'nodes_data', '%s.tar.gz' % node['name'])
+        if os.path.exists(node_backup_file):
```

Comment:
https://github.com/Netcracker/KubeMarine/blob/main/documentation/Maintenance.md#restore-procedure
> Everything should be the same, down to the names and addresses of the nodes, their amounts and roles

If the requirement is satisfied, the file `node_backup_file` always exists. Otherwise, we need to throw exception, and do no silently ignore the missed file.

Reply:
In that case, can we raise a FileNotFoundError exception with a message. 
 Like below 
 

>              result = node['connection'].sudo(restore_command)
>             cluster.log.debug(result)
>         else:
>             raise FileNotFoundError(f'Backup file for node \'{node["name"]}\' not found')

Reply:
Ok, but it is a separate issue not related to your original PR. So please provide additional Test Case with
* *Result Before: something*
* *Result After: FileNotFoundError*